docs/_docs/04_reference/cli-options.md

```diff
@@ -464,6 +465,46 @@ Example:
 }
 ```
 
+### output-module-groups
+
+Output groups of modules ordered for apply as a list of list in JSON (useful for CI use cases).
+
+Example:
+
+```bash
+terragrunt output-module-groups
+```
+
+This will recursively search the current working directory for any folders that contain Terragrunt modules and build
+the dependency graph based on [`dependency`](/docs/reference/config-blocks-and-attributes/#dependency) and
+[`dependencies`](/docs/reference/config-blocks-and-attributes/#dependencies) blocks. This may produce output such as:
+
+```
+[
```

Comment:
I was wondering if json format should include group key, to be consistent with text output, like:

{
"Group 1": [
  "stage/frontend-app"
],
"Group 2": [
    "mgmt/bastion-host",
    "stage/search-app"
]
...

}

Reply:
Done :) Can I have another review @denis256  ?